README.md

```diff
@@ -16,6 +16,39 @@
 [ci-badge]: https://github.com/adazzle/react-data-grid/workflows/CI/badge.svg
 [ci-url]: https://github.com/adazzle/react-data-grid/actions
 
+## Features
+
+- [React 16.14+ & 17.0+](package.json) support
+- [Evergreen browsers and server-side rendering](browserslist) support
```

Comment:
Not sure if RDG support server side rendering. I thing `useLayoutEffect` may show a warning